tests/materialize/materialize-databricks/checks.sh

```diff
@@ -0,0 +1,2 @@
+# cleanup the snapshot by replacing uuids with a placeholder so that the test is reproducible
+sed -i '' 's/.\{36\}\.json/<uuid>/g' ${SNAPSHOT}
```

Comment:
Would it make sense to include this in the chain here? https://github.com/estuary/connectors/blob/f282c882c66de33204c54ede7950641ea19b3df2/tests/materialize/materialize-databricks/fetch.sh#L7-L9